packages/point-intercept/src/__tests__/main.test.js

```diff
@@ -0,0 +1,142 @@
+import * as React from 'react';
+import { PlotPoints } from '@pie-lib/charting';
+import { shallowChild } from '@pie-lib/test-utils';
+import Controls from '../controls';
+import CorrectAnswerToggle from '@pie-lib/correct-answer-toggle';
+import { Feedback } from '@pie-lib/render-ui';
+import Main from '../main';
+
+describe('Main', () => {
```

Comment:
there'd be a couple of other functions we could test? `deleteSelection`, `movePoint`?

Reply:
They are tested already. 
You can find the relevant tests -

here: https://github.com/pie-framework/pie-ui/pull/14/files#diff-88f2e0a2357f6a20ba964c8e946d5d1dR85
and here: https://github.com/pie-framework/pie-ui/pull/14/files#diff-88f2e0a2357f6a20ba964c8e946d5d1dR108

I think I got pretty good coverage here for the internal functionality.